app.js

```diff
@@ -81,28 +90,60 @@ var imageConstructorArray = [
   shark,
   sweep,
   tauntaun,
-  thumbs,
   unicorn,
   usb,
   waterCan,
   wineGlass,
-] ;
+];
+var percentArray = [];
+var threePic = [];
+var oldPics = [];
+var numOfPics = 3;
 // populates three images pulled at an index from the array. the index is a random number
 function populateThree(){
-  for (var i = 0; i < 3; i++) {
+  oldPics = threePic;
+  threePic = [];
+  while (threePic.length < numOfPics) {
     var integerIndex = getRandomInt();
-    //this makes the number we generated storeable since calling the functon just makes a new number
-    imageConstructorArray[integerIndex].addPicture(integerIndex);
+    if (threePic.indexOf(integerIndex) === -1 && oldPics.indexOf(integerIndex) === -1) {
```

Comment:
Since we worked on this together, you should write comments to yourself so that you remember how it works.